CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 # redisio
 
+## 4.1.1 (2020-08-14)
```

Comment:
We typically wait to tag this when we actually publish the cookbook.
```suggestion
## Unreleased
```

Reply:
Thanks for the headsup, I committed your suggestion.